superbuild/projects_modules/qwt.cmake

```diff
@@ -46,7 +48,7 @@ set(cmake_args
   -DQT_QMAKE_EXECUTABLE:FILEPATH=${QT_QMAKE_EXECUTABLE}
 )
 
-ep_GeneratePatchCommand(${ep} QWT_PATCH_COMMAND qwt-6.3.patch)
+#ep_GeneratePatchCommand(${ep} QWT_PATCH_COMMAND qwt-6.3-svn.patch)
```

Comment:
you could have kept this line, and ${QWT_PATCH_COMMAND} line 84 then?

Reply:
This one works only on a git repo.

Reply:
oki